ext/symengine/symengine.h

```diff
@@ -11,5 +11,6 @@ VALUE c_basic;
 VALUE c_symbol;
 VALUE c_integer;
 VALUE c_rational;
+VALUE c_constants;
```

Comment:
Yes, right now `E` is an object of type `Basic` which is fine, but you can make a new class `c_constants` and make that the object type of `E`